test/e2e_node/summary_test.go

```diff
@@ -370,6 +370,20 @@ var _ = SIGDescribe("Summary API", framework.WithNodeConformance(), func() {
 
 	framework.Context("when querying /stats/summary under pressure", feature.KubeletPSI, framework.WithSerial(), func() {
```

Comment:
I would suggest an alternative solution for cgroupv1 problem. In test framework we can add a flag `--fail-cgroupv1=true` unless otherwise is not specified in the test's kubelet arguments. This way test will not even start on cgroupv1 when intended for cgroupv2. 

Also tests requiring cgroupv1 can be marked with the `Feature.Cgroupv1` so jobs can filter them in and out.

Reply:
/hold